content/en/blog/2025/ambient-multicluster/index.md

```diff
@@ -0,0 +1,100 @@
+and addressing pods by namespaced name rather than IP eliminates issues with overlapping IP spaces.
+Together, these design choices enable cross-cluster connectivity without changing cluster networking or restarting workloads,
+even as clusters are added or removed.
+
+### Double HBONE
+
+Ambient multicluster uses nested [HBONE](https://istio.io/latest/docs/ambient/architecture/hbone/) connections to efficiently secure traffic traversing cluster boundaries.
+An outer HBONE connection encrypts traffic to the east-west gateway and allows the source ztunnel and east-west gateway to verify each other's identity.
+An inner HBONE connection encrypts traffic end-to-end, which allows the source ztunnel and destination ztunnel to verify each other's identity.
+At the same time, the HBONE layers allow ztunnel to effectively reuse cross-cluster connections, minimizing TLS handshakes.
+
+{{< image link="./mc-ambient-traffic-flow.png" caption="Istio ambient multicluster traffic flow" >}}
+
+### Service Discovery and Scope
+
+Marking a service global enables cross-cluster communication.
+Istiod configures east-west gateways to accept and route global service traffic to local pods and
+programs ztunnels to load balance global service traffic to remote clusters.
+
+Mesh administrators define the label-based criteria for global services via the `ServiceScope` API,
+and app developers label their services accordingly.
+The default `ServiceScope` is
+
+{{< text yaml >}}
+serviceScopeConfigs:
+  - servicesSelector:
+      matchExpressions:
+        - key: istio.io/global
+          operator: In
+          values: ["true"]
+    scope: GLOBAL
+{{< /text >}}
+
+meaning that any service with the `istio.io/global=true` label is global.
+Although the default value is straightforward, the API is flexible and can express complex conditions using a mix of ANDs and ORs.
+
+By default, ztunnel load balances traffic uniformly across all endpoints --even remote ones--,
+but is configurable through the service's `trafficDistribution` field to only cross cluster boundaries when there are no local endpoints.
+Thus, users have control over whether and when traffic crosses cluster boundaries with no changes to application code.
+
+## Limitations and Roadmap
+
+Although the current implementation of ambient multicluster has the foundational features for a multicluster solution,
+there is still a lot of work to be done.
+We are looking to improve the following areas
+
+* Service and waypoint configuration must be uniform across all clusters.
+* No cross-cluster L7 failover (L7 policy is applied at the destination cluster).
+* No support for direct pod addressing or headless services.
+* Support only for multi-primary deployment model.
+* Support only for one network per cluster deployment model.
+
+We are also looking to improve our reference documentation, guides, testing, and performance.
+
+If you would like to try out ambient multicluster, please follow [this guide](TODO).
```

Comment:
Just making a note that this will need to b e changed before merging. I think it's waiting on #16709